### DISCUSSION TITLE
v5.0 released!

### DISCUSSION THREAD

**Discussion by axelboc** [axelboc/anki-ultimate-geography#475] [New releases]
- **[Release notes](https://github.com/anki-geo/ultimate-geography/releases/tag/v5.0)**
- **[Upgrade instructions](https://github.com/anki-geo/ultimate-geography/wiki/Upgrade-instructions#from-v41-to-v50)**

If you encounter any issues upgrading to the new version, please report them in this discussion thread. Enjoy!

**Comment by KobaYounes** [comment]
Hello, I tried upgrading from the french extended 4.1.1 version. Following the instructions I loose progress.

**Reply by axelboc** [reply]
Sorry to hear that 😢 

I've just tried the upgrade process again as follows and my progress was kept, so I have no idea what could be going wrong:

1. Create a new Anki profile.
2. Import the standard French 4.1.1 deck.
3. Review a few cards.
4. Run the French GUID replacement script.
5. Import the standard French v5 deck.

Did you make any sorts of customisations to the deck? Are you using the extended deck maybe?

**Reply by jeprecated** [reply]
@KobaYounes "Loose progress" in what way? All notes are now new? Only some progress is lost?

**Reply by aplaice** [reply]
The process should (and from my tests both previously and now does) run for the extended deck as well.

To add to the barrage of questions: could you check the total number of cards in your "Ultimate Geography" deck? (After selecting the "Ultimate Geography" deck press <kbd>t</kbd> to get to the statistics.) (Is it 1524 (or 1526 with two now-empty cards)?)

**Reply by KobaYounes** [reply]
@ohare93 Yes all notes are new.
@aplaice No customisations at all. I will try doing what you said and get back at you in couple minutes.

**Reply by KobaYounes** [reply]
After running the GUID migration script I'm supposed to follow this?

> Major version
Major versions (e.g. v3.0) typically indicate that upgrading may lead to a significant loss of progress, or to the loss of some of the customisations you may have made to the deck that would normally be preserved. This can occur, for instance, when significant changes are made to the structure of the deck.

Read the release notes carefully to know if the loss of progress/customisation actually concerns you. If it does, the release notes will tell you how best to upgrade and may even offer you ways to preserve or mitigate your losses. Worst-case scenario, you may have no other choice but to accept the losses and perform a "clean import" as follows:

Open Anki and make sure your devices are all synchronised.
Delete the Ultimate Geography deck.
In the Tools menu, select Manage Note Types, then delete the Ultimate Geography note type.
In the Tools menu, select Check Database.
Sync the changes with AnkiWeb and with all your devices.
You can now follow the steps of the Getting started section to install the major version.

**Reply by KobaYounes** [reply]
Because If I just import from disk using crowdank after running the script I end up with 1526 cards and don't loose any progress.

**Reply by aplaice** [reply]
Thanks for replying and describing the situation in detail!

<hr/>

The procedures regarding the clean import are supposed to be a worst-case scenario, if nothing else works, so in general it's best if you don't follow them. I can see, however, how you could find the current wording misleading. Sorry! (Perhaps we should re-word it? (Should we add " (usually not necessary!)" after "Worst-case scenario" for extra emphasis or re-arrange the section more extensively?))

> Because If I just import from disk using crowdank after running the script I end up with 1526 cards and don't loose any progress.

Yes, importing from disk using crowdanki after running the script was the intended upgrade path. I'm glad that you don't lose progress, in this case!

**Reply by axelboc** [reply]
I've made an attempt at clarifying things here: #479 - let me know what you think!

**Comment by KobaYounes** [comment]
Thank you all for your help and for this awesome deck.
